src/dxtbx/imageset.py

```diff
@@ -55,7 +57,12 @@ def _expand_template(template: str, indices: Iterable[int]) -> list[str]:
     pfx = template.split("#")[0]
     sfx = template.split("#")[-1]
     count = template.count("#")
-    return [f"{pfx}{index:0{count}}{sfx}" for index in indices]
+    if count == 1:
+        # https://github.com/cctbx/dxtbx/issues/646
```

Comment:
We sure we want to keep an issue reference in the code, or better to explain in a two line comment what we are doing here?